### ISSUE TITLE
login/out flash message - timed

### ISSUE THREAD

**Issue by RyanTG** [scottwainstock/pbm#476]
When you're looking at a map and then you log out, the message is in a weird place. I can either fix this, or we could make it similar to the flash (error) message that comes up when editing location metadata: "Phone format invalid, please use ###-###-####." That is, can we have it pop up and then disappear after five seconds? I don't see any reason why it needs to stay there forever.

**Comment by RyanTG** [comment]
I think this is the main thing left on the website for the auth update. The "you just logged in/out" message stays there forever, since our site never refreshes.

Remember when there were TWO messages showing up, and I said, "get rid of one of them, and let's use the other"? Not sure if the one we got rid of was the one that's capable of being timed, while the one we kept isn't. If that's the case, then we should swap it.

I think this is what we removed. https://github.com/scottwainstock/pbm/commit/3cc3303b05bd2fe78a822b40a787f353ef096181
